services/libs/aster-std/src/fs/utils/inode.rs

```diff
@@ -245,6 +251,8 @@ pub trait Inode: Any + Sync + Send {
 
     fn set_mode(&self, mode: InodeMode);
 
+    fn set_owner(&self, uid: Uid, gid: Gid);
```

Comment:
I think this interface is ill-defined. The `chown` system call may change either of the owner and group of a file, or both. The interface only supports changing both. So I think both methods should be added, one is `set_owner`, the other is `set_group`.

Reply:
Yes